src/Collection/Map.php

```diff
@@ -185,4 +186,69 @@ public function iterator(): Iterator
     {
         return new MapIterator($this->map);
     }
+
+    /**
+     * @return T[]
+     */
+    public function values(): array
```

Comment:
This is interesting. The question here is: should we return `Sequence` instead of native array? :thinking:

Reply:
Yup, `Sequence` is not implemented right now ... so maybe just open new issue?